asset/src/helpers.ts

```diff
@@ -0,0 +1,190 @@
+// @ts-ignore
+import parseError from '@terascope/error-parser';
```

Comment:
Use `parseError` from `@terascope/utils`

```suggestion
import { parseError } from '@terascope/utils';
```